### PR TITLE
chore(docs): Issue Templates

### PR DESCRIPTION
**Pull request by 1nb0und**
Introducing issue templates for easier triaging and auto labeling. Inspired from Zeebe's issue templates.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+```
+
+</p>
+</details>
+
+**Environment:**
+- OS: <!-- [e.g. Linux] -->
+- Deployment: <!-- [SaaS or SM (Self Managed)] --> 
+- Zeebe Version: <!-- [e.g. 0.20.0] -->
+- SpringBoot Version: <!-- [e.g. 2.7.x, 3.x.x] -->
```
Comment (1nb0und) [inline .github/ISSUE_TEMPLATE/bug_report.md:46]: These 2 might be clumped under `Configuration` or can be left as standalone. Could also add JDK version as well but that might be a too much as this list will be growing

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+<STACKTRACE>
+```
+
+</p>
+</details>
+
+**Environment:**
+- OS: <!-- [e.g. Linux] -->
+- Deployment: <!-- [SaaS or SM (Self Managed)] --> 
+- Zeebe Version: <!-- [e.g. 0.20.0] -->
```
Comment (berndruecker) [inline .github/ISSUE_TEMPLATE/bug_report.md:45]: Isn't spring-zeebe version also important? 
And my hope would be that Zeebe Client in itself is transitive - but might differ.
Zeebe "server" might not be too interesting I guess - more of a corner case?

**Review by berndruecker — APPROVED** [review]
Yeah - why not. I would say just go ahead and improve on the fly

**Review by chillleader — APPROVED** [review]
Looks great!

**Pull request merged**